### PR TITLE
feat:add default no tracking and default table name pluralizing for EFCore

### PR DESCRIPTION
**Pull request by MapleWithoutWords**

### CODE REVIEW

File: src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -79,10 +80,10 @@ public static class ServiceCollectionExtensions
     private static MasaDbContextOptions<TDbContextImplementation> CreateMasaDbContextOptions<TDbContextImplementation>(
         IServiceProvider serviceProvider,
         Action<IServiceProvider, MasaDbContextOptionsBuilder>? optionsBuilder,
-        bool enableSoftDelete)
+        bool enableSoftDelete, bool enablePluralizingTableName)
```
Comment (zhenlei520) [inline src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/Extensions/ServiceCollectionExtensions.cs:83]: new line

File: src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/MasaDbContext.cs [hunk 2]
```diff
@@ -44,6 +44,7 @@ public abstract class MasaDbContext : DbContext, IMasaDbContext
     protected MasaDbContext(MasaDbContextOptions options) : base(options)
     {
         Options = options;
+        this.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
```
Comment (zhenlei520) [inline src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/MasaDbContext.cs:47]: Replace this or to base

File: src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/MasaDbContext.cs [hunk 3]
```diff
@@ -73,6 +74,14 @@ protected sealed override void OnModelCreating(ModelBuilder modelBuilder)
 
         foreach (var provider in Options.ModelCreatingProviders)
             provider.Configure(modelBuilder);
+
+        if (Options.EnablePluralizingTableName)
```
Comment (zhenlei520) [inline src/Contrib/Data/Orm/EFCore/Masa.Contrib.Data.EFCore/MasaDbContext.cs:78]: Turn on plurals?

File: src/Contrib/StackSdks/Masa.Contrib.StackSdks.Tsc/Tracing/Handler/HttpClientInstrumentHandler.cs [hunk 4]
```diff
@@ -13,6 +13,7 @@ public virtual async void OnHttpRequestMessage(Activity activity, HttpRequestMes
 
     public virtual void OnHttpResponseMessage(Activity activity, HttpResponseMessage httpResponseMessage)
     {
+        Console.WriteLine($"HttpClient's TraceId={activity?.TraceId}");
```
Comment (zhenlei520) [inline src/Contrib/StackSdks/Masa.Contrib.StackSdks.Tsc/Tracing/Handler/HttpClientInstrumentHandler.cs:16]: ?

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]

**Comment by doddgu** [comment]
@MapleWithoutWords Resole conflicts

**Review by zhenlei520 — APPROVED** [review]
LGTM @doddgu

**Review by doddgu — APPROVED** [review]

**Pull request merged**